Tasks/ExtractFilesV1/task.json

```diff
@@ -58,6 +58,14 @@
             "required": true,
             "defaultValue": "false",
             "helpMarkDown": "Select this option to overwrite existing files in the destination directory."
+        },
+        {
+            "name": "pathToSevenZipTool",
+            "type": "string",
+            "defaultValue": "",
```

Comment:
I think it makes sense to set the default value since it was introduced in code.

Reply:
I don't think so since default value in code is introducing only for windows and it is not actual for MacOS and Linux.